liquidcore/home/views.py

```diff
@@ -11,6 +11,7 @@ def homepage(request):
         'liquid_apps': settings.LIQUID_APPS,
         'liquid_version': settings.LIQUID_VERSION,
         'liquid_core_version': settings.LIQUID_CORE_VERSION,
+        '2fa_enabled': settings.LIQUID_2FA
```

Comment:
python coding style requires a trailing comma here, in case you want to shuffle lines around